types/relay-runtime/lib/store/RelayStoreTypes.d.ts

```diff
@@ -416,88 +416,143 @@ interface OperationDescriptor {
+    | Readonly<{
+        name: 'network.next';
+        networkRequestId: number;
+        response: GraphQLResponse;
+    }>
+    | Readonly<{
+        name: 'network.error';
+        networkRequestId: number;
+        error: Error;
+    }>
+    | Readonly<{
+        name: 'network.complete';
+        networkRequestId: number;
+    }>
+    | Readonly<{
+        name: 'network.unsubscribe';
+        networkRequestId: number;
+    }>
     | Readonly<{
-          name: 'execute.info';
-          transactionID: number;
-          info: any;
-      }>
+        name: 'execute.start';
+        executeId: number;
+        params: RequestParameters;
+        variables: Variables;
+        cacheConfig: CacheConfig;
+    }>
     | Readonly<{
-          name: 'network.info';
-          transactionID: number;
-          info: unknown;
-      }>
+        name: "execute.next";
+        executeId: number;
+        response: GraphQLResponse;
+        duration: number;
+    }>
     | Readonly<{
-          name: 'network.start';
-          transactionID: number;
-          params: RequestParameters;
-          variables: Variables;
-      }>
+        name: 'execute.async.module';
+        executeId: number;
+        operationName: string;
+        duration: number;
+    }>
     | Readonly<{
-          name: 'network.next';
-          transactionID: number;
-          response: GraphQLResponse;
-      }>
+        name: 'execute.flight.payload_deserialize';
+        executeId: number;
+        operationName: string;
+        duration: number;
+    }>
     | Readonly<{
-          name: 'network.error';
-          transactionID: number;
-          error: Error;
-      }>
+        name: 'execute.error';
+        executeId: number;
+        error: Error;
+    }>
     | Readonly<{
-          name: 'network.complete';
-          transactionID: number;
-      }>
+        name: 'execute.complete';
+        executeId: number;
+    }>
     | Readonly<{
-          name: 'network.unsubscribe';
-          transactionID: number;
-      }>
+        name: 'store.publish';
+        source: RecordSource;
+        optimistic: boolean;
+    }>
     | Readonly<{
-          name: 'store.publish';
-          source: RecordSource;
-          optimistic: boolean;
-      }>
+        name: 'store.snapshot';
+    }>
     | Readonly<{
-          name: 'store.snapshot';
-      }>
+        name: 'store.restore';
+    }>
     | Readonly<{
-          name: 'store.restore';
-      }>
+        name: 'store.gc';
+        references: Set<DataID>;
+    }>
     | Readonly<{
-          name: 'store.gc';
-          references: Set<DataID>;
-      }>
+        name: 'store.notify.start';
+        sourceOperation?: OperationDescriptor | undefined;
+    }>
     | Readonly<{
-          name: 'store.notify.start';
-      }>
+        name: 'store.notify.complete';
+        sourceOperation?: OperationDescriptor | undefined;
+        updatedRecordIDs: UpdatedRecords;
+        invalidatedRecordIDs: Set<DataID>;
```

Comment:
In flow, these 2 are both typed the same as `DataIDSet`, which we previously typed as `Set<DataID>;`.

Should we also have these typed the same, or is `UpdatedRecords` type better?

Reply:
Nah nah, super good! Great pickup.